runtime/tests/jvmtitests/src/com/ibm/jvmti/tests/eventSampledObjectAllocation/soae001.c

```diff
@@ -0,0 +1,122 @@
+/*******************************************************************************
```

Comment:
Again, maybe remove "event" from the description.

Reply:
Same change made here.